CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project (loosely) adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.2.0 - 2024-03-13
+
+### Added
+
+- `SDJsonpath` and `SDJWTDisclosureStruct` from `@meeco/sd-jwt`
```

Comment:
should this be 

- Updated `@meeco/sd-jwt` dependency, which now includes `SDJsonpath` and `SDJWTDisclosureStruct`

Reply:
would it be good idea to update major version
this is because client code might directly use functions like `decodeDisclosures` and `VerifySDJWTOptions`.

Reply:
there were other minor dev dependencies updated, reflected in the `package-lock.json`

> 
> would it be good idea to update major version
> this is because client code might directly use functions like decodeDisclosures and VerifySDJWTOptions.
No, those functions are not exposed by this package. So there's no need for a major version update.
The `@meeeco/sd-jwt` lib did have a major version change